NT LP "Svalinn"

Firefox13 opened the pull request:
## About The Pull Request

This PR adds a new NT branded laser pistol "Svalinn". Small-sized pistol, accepts S-class cells, consumes 50 per shot, deals 30(standard laser projectile) burn per shot, fits in holsters and pockets. Basically NT-branded Martin pistol, minus the stun feature, but with charge mode.
This PR adds the Svalinn to:
gun drops from trash
autolathe disks
NT Energy Pack(The price was adjusted accordingly)
Preacher, Custodian, Acolyte closets(agrolytes don't need to shoot plants)

## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->
![изображение](https://user-images.githubusercontent.com/57810301/90158439-a9fdbc80-dd97-11ea-9b29-9a4e85c4d83d.png)
(most of the) NT gets it's own sidearm, albeit somewhat weak, just like Martin. Also, diversity.

## Changelog
:cl: Firefox13, Armolitsky
add: Added NT LP "Svalinn", new NeoTheology brand laser pistol. Preacher, Acolytes and Custodians lockers are now supplemented with one, as well as NT Energy Pack comes with two laser pistols.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

seekin4u commented:
`
code/datums/autolathe/guns.dm:195:error: /obj/item/weapon/gun/energy/ntpistol: undefined type path
code/modules/cargo/packs.dm:180:error: /obj/item/weapon/gun/energy/ntpistol: undefined type path
code/modules/cargo/packs.dm:181:error: /obj/item/weapon/gun/energy/ntpistol: undefined type path
code/game/objects/items/weapons/autolathe_disks.dm:1028:error: /obj/item/weapon/gun/energy/ntpistol: undefined type path
code/game/objects/structures/crates_lockers/closets/job_closets.dm:114:error: /obj/item/weapon/gun/energy/ntpistol: undefined type path
code/game/objects/structures/crates_lockers/closets/job_closets.dm:138:error: /obj/item/weapon/gun/energy/ntpistol: undefined type path
code/game/objects/structures/crates_lockers/closets/secure/chaplain.dm:34:error: /obj/item/weapon/gun/energy/ntpistol: undefined type path
code/game/objects/random/guns.dm:16:error: /obj/item/weapon/gun/energy/ntpistol: undefined type path
`

Firefox13 commented:
> 
> 
> `code/datums/autolathe/guns.dm:195:error: /obj/item/weapon/gun/energy/ntpistol: undefined type path code/modules/cargo/packs.dm:180:error: /obj/item/weapon/gun/energy/ntpistol: undefined type path code/modules/cargo/packs.dm:181:error: /obj/item/weapon/gun/energy/ntpistol: undefined type path code/game/objects/items/weapons/autolathe_disks.dm:1028:error: /obj/item/weapon/gun/energy/ntpistol: undefined type path code/game/objects/structures/crates_lockers/closets/job_closets.dm:114:error: /obj/item/weapon/gun/energy/ntpistol: undefined type path code/game/objects/structures/crates_lockers/closets/job_closets.dm:138:error: /obj/item/weapon/gun/energy/ntpistol: undefined type path code/game/objects/structures/crates_lockers/closets/secure/chaplain.dm:34:error: /obj/item/weapon/gun/energy/ntpistol: undefined type path code/game/objects/random/guns.dm:16:error: /obj/item/weapon/gun/energy/ntpistol: undefined type path`

Should've included .dme edits too, since I've added a new file.
Fixed

SpadesSlick commented on code/game/objects/items/weapons/autolathe_disks.dm:
```diff
@@ -1019,6 +1019,16 @@
 		/datum/design/autolathe/cell/small/high,
 	)
 
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/guns/nt_counselor
```
Please keep all names consistent with actual gun name, make it ``nt_svalinn``.

SpadesSlick commented on code/datums/autolathe/guns.dm:
```diff
@@ -190,6 +190,10 @@
 	name = "FS PDW E \"Martin\""
 	build_path = /obj/item/weapon/gun/energy/gun/martin
 
+/datum/design/autolathe/gun/ntpistol
```
Same as above.

SpadesSlick commented on code/game/objects/random/lathe_disks.dm:
```diff
@@ -38,6 +38,7 @@
 				/obj/item/weapon/computer_hardware/hard_drive/portable/design/guns/nt_counselor = 2,
 				/obj/item/weapon/computer_hardware/hard_drive/portable/design/guns/fs_spiderrose = 1,
 				/obj/item/weapon/computer_hardware/hard_drive/portable/design/guns/fs_martin = 3,
+				/obj/item/weapon/computer_hardware/hard_drive/portable/design/guns/nt_counselor = 3,
```
Same as above.

SpadesSlick commented on code/modules/projectiles/guns/energy/ntpistol.dm:
```diff
@@ -0,0 +1,24 @@
+/obj/item/weapon/gun/energy/ntpistol
```
Same as above, also change a file name.

GrayRachnid commented:
Added the rebalance tag since this spawns it in NT lockers. Should be taken into consideration.
